tests/compiler/typealias.ts

```diff
@@ -1,8 +1,9 @@
-export type alias = i32;
+type A = f32;
```

Comment:
Why you removed `export`? I should be preserved

Reply:
I added it back and changed the names to match the TODO.